WebProfilerServiceProvider.php

```diff
@@ -379,6 +379,7 @@ public function subscribe(Container $app, EventDispatcherInterface $dispatcher)
         $dispatcher->addSubscriber($app['profiler']->get('request'));
 
         if (isset($app['var_dumper.data_collector'])) {
+            $app['var_dumper.dump_listener']->configure();
             $dispatcher->addSubscriber($app['var_dumper.dump_listener']);
```

Comment:
This means this line is useless, isn't it? the effect of this patch is to register the dump() handle earier, thus there is no need to register any subscriber for later config, this is already done with the line above. Or did I miss something?

Reply:
@qkdreyer would you mind removing this line please?

Reply:
fixed in 9657191